### ISSUE TITLE
Are there any Caffe models pretrained with full dropout (every neuron) available for download?

### ISSUE THREAD

**Issue by awadsb1** [yaringal/DropoutUncertaintyCaffeModels#4]
This would be helpful as I'm trying to use for a research project.  Specifically, I'm hoping for a CNN trained with a large benchmark image set (ILSVRC or COCO...etc.).  Any pre-trained weights (models) available for something like that?
